server/build.gradle

```diff
@@ -125,15 +125,13 @@ jib {
     to {
         if (project.hasProperty("test-tc-system")) {
             image = "231168606641.dkr.ecr.us-east-1.amazonaws.com/test-ecs"
-        } else {
+        } else if (project.hasProperty("prod-tc-system")) {
             image = "968457613372.dkr.ecr.us-east-1.amazonaws.com/tbbtalentv2"
         }
         credHelper = 'ecr-login'
     }
 }
 
-tasks.build.dependsOn tasks.jib
```

Comment:
Will jib run if this is not present?

Reply:
Jib will not run automatically when executing "gradle build"

It has to be called explicitly now - "gradle jib" - this prevents it from running when not required for local builds.

Of course for Git Action builds we want Jib to always run - therefore it must be present as an explicit step. The below PR adds the required step to the Git Actions:

https://github.com/Talent-Catalog/talentcatalog/pull/250